src/main/resources/actions.sql

```diff
@@ -0,0 +1,20 @@
+create table filters_contingency_list (
```

Comment:
did you create this with the
```
spring.jpa.properties.javax.persistence.schema-generation.scripts.action=create
spring.jpa.properties.javax.persistence.schema-generation.scripts.create-target=create.sql
spring.jpa.properties.javax.persistence.schema-generation.scripts.create-source=metadata
```
(or equivalent) ?

Reply:
I created the schema with property 'spring.jpa.hibernate.ddl-auto' set to 'create' in application.yaml file